### ISSUE TITLE
Othere databases/API to look at/include

### ISSUE THREAD

**Issue by adam3smith** [PESData/awardFindR#1]
Running list:

## Government funders
- [x] https://federalreporter.nih.gov/
- [x] https://ies.ed.gov/funding/grantsearch/index.asp?mode=1&sort=1&order=1&searchvals=data&SearchType=or&checktitle=on&checkaffiliation=on&checkprincipal=on&checkquestion=on&checkprogram=on&checkawardnumber=on&slctAffiliation=0&slctPrincipal=0&slctYear=0&slctProgram=0&slctGoal=0&slctCenter=0&FundType=1&FundType=2
- [x] https://securegrants.neh.gov/publicquery/main.aspx
- [x] Federal Reporter 
- [x] USA Spending

## Private funders
- [x] https://www.openphilanthropy.org/giving/grants
- [x] https://sloan.org/grants-database
- [x] Social Science Research Council
- [x] Robert Wood Johnson Foundation
- [x] John Templeton Foundation
- [x] Bill and Melinda Gates Foundation
- [x] Andrew Mellon Foundation
- [x] MacArthur Foundation
- [x] Carnegie Foundation
- [x] Open Society Foundations
- [x] Rockefeller Foundation
- [x] Russell Sage Foundation
- [x] Arnold Foundation

**Comment by mccallc** [comment]
Focusing on NIH first; beginning NIH integration with commit https://github.com/PESData/sciscrape/commit/97bd35d9b6084119762b1cb191be8dbcc2deb416

**Comment by mccallc** [comment]
USAspending implemented fully as of cb2f6a0c23ce27c950f2d2df5dac5b40f626b0e8
Agencies have to be specified manually. Currently pulls:
- IMLS
- Smithsonian institution
- Depts of Commerce, education and the interior

AHRQ, HHS, NASA, EPA, DOD, VA and USDA included through the federal reporter module

Potential direct sources?
Dept of Energy: https://pamspublic.science.energy.gov/WebPAMSExternal/interface/awards/AwardSearchExternal.aspx
NOAA: https://www.noaa.gov/office-education/elp/grants/awards

**Comment by adam3smith** [comment]
IMLS also has an individual grant search that looks great: https://www.imls.gov/grants/awarded-grants -- you can just download the PDF with all relevant grants based on a query

**Comment by adam3smith** [comment]
Other private funders to look into (next priorities in bold)

- American Council of Learned Societies (ACLS)
- **Arcadia Fund** 
- Aspen Institute
- Commonwealth Fund
- Eric & Wendy Schmidt Fund for Strategic Innovation 
- Ewing Marion Kauffman Foundation
- **Ford Foundation**
- **Gordon and Betty Moore Foundation** 
- James S. McDonnell Foundation
- Howard Hughes Medical Institute
- Leona M. and Harry B. Helmsley Charitable Trust
- Lumina Foundation
- Templeton World Charity Foundation
- **William T. Grant Foundation**

**Comment by mccallc** [comment]
> IMLS also has an individual grant search that looks great: https://www.imls.gov/grants/awarded-grants -- you can just download the PDF with all relevant grants based on a query

Yeah looks like that site uses undocumented API at awsgov-api.imls.gov to generate a csv file, will implement
